web/themes/custom/cloudy/templates/views/views-view-field--field-youtube.html.twig

```diff
@@ -0,0 +1,31 @@
+{#
+/**
+ * @file
+ * Theme override for a single field in a view.
+ *
+ * Available variables:
+ * - view: The view that the field belongs to.
+ * - field: The field handler that can process the input.
+ * - row: The raw result of the database query that generated this field.
+ * - output: The processed output that will normally be used.
+ *
+ * When fetching output from the row this construct should be used:
+ * data = row[field.field_alias]
+ *
+ * The above will guarantee that you'll always get the correct data, regardless
+ * of any changes in the aliasing that might happen if the view is modified.
+ *
+ * @see template_preprocess_views_view_field()
+ */
+#}
+{% if output %}
+  {% include '@components/link/link.twig' with {
+    icon: {
+      name: 'youtube',
+      fontAwesome: true,
+      brand: true
+    },
+    text: output,
+    url: "http://youtube.com/#{output}"
```

Comment:
Review current youtube field in production. I think the contacts field is a link and the deprecated field is a username field.